bot/client.js

```diff
@@ -57,4 +57,15 @@ client.on('message', (message) => {
   }
 });
 
+client.on('guildMemberAdd', (member) => {
+  // Send dm to user
+  member.send(`Welcome to our server ${member}!`);
+  // Send message to general channel
+  const channel = member.guild.channels.find('name', 'general');
+  if (!channel) {
+    return;
```

Comment:
Erroring silently is not a good scenario. at least kick back to the debugger or craft a better message to respond to the user with, though in this case that would not be appropriate as a user did not kick off this interaction.